pkg/solidity-utils/test/foundry/TransientEnumerableSet.t.sol

```diff
@@ -0,0 +1,233 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.24;
+
+import "forge-std/Test.sol";
+
+import { TransientEnumerableSet } from "../../contracts/openzeppelin/TransientEnumerableSet.sol";
+
+contract TransientEnumerableSetTest is Test {
+    using TransientEnumerableSet for TransientEnumerableSet.AddressSet;
+
+    TransientEnumerableSet.AddressSet private testSet;
+    address internal constant ADDRESS_ONE = address(1);
+    address internal constant ADDRESS_TWO = address(2);
+    address internal constant ADDRESS_THREE = address(3);
+    address internal constant NON_EXISTENT_ADDRESS = address(10);
+
+    function setUp() public virtual {
+        testSet.add(ADDRESS_ONE);
+    }
+
+    function testContains() public view {
+        assertFalse(testSet.contains(ADDRESS_ONE), "Address added in another transaction found in testSet");
```

Comment:
Ha, I didn't think about this detail. Can we add a comment in the setup explaining that the value is expected to be erased afterwards because `setUp` is executed in a different transaction?